contract-rs/src/lib.rs

```diff
@@ -83,10 +90,16 @@ impl TransferFaucet {
         //     Computing hash of the message
         let hash = env::sha256(&message);
         //     Checking that the resulting hash has enough leading zeros.
-        assert!(num_leading_zeros(&hash) >= self.min_difficulty, "The proof is work is too weak");
+        assert!(
+            num_leading_zeros(&hash) >= self.min_difficulty,
+            "The proof is work is too weak"
+        );
 
         // Checking that the given hash is not used yet and remembering it.
-        assert!(!self.existing_hashes.insert(&hash), "The given hash is already used for transfer");
+        assert!(
+            !self.existing_hashes.insert(&hash),
```

Comment:
```suggestion
            self.existing_hashes.insert(&hash),
```
`near-sdk` at some version has a fix makes `insert` returns `true` as described in the comment

Reply:
Great, thanks!